src/components/timeline/TrackScreenshots.js

```diff
@@ -220,19 +225,24 @@ class HoverPreview extends PureComponent<HoverPreviewProps> {
 
```

Comment:
Does rounding make a big difference, why we are rounding them?

Reply:
This is about pixels: if it's not rounded, it means this will stop "between" pixels, and we can have a blurry edge.
In today's web state, we should probably take the dpi into account (something like `Math.floor(value * dpi) / dpi)`), but this is IMO overcomplicated for our case here.